scripts/qa_crowbarsetup.sh

```diff
@@ -3395,11 +3396,21 @@ function oncontroller_testsetup()
     nova stop "$instanceid"
     wait_for 100 1 "test \"x\$(nova show \"$instanceid\" | perl -ne 'm/ status [ |]*([a-zA-Z]+)/ && print \$1')\" == xSHUTOFF" "testvm to stop"
 
+    # check that no port is in binding_failed state
+    for p in $(neutron port-list -f csv -c id --quote none | grep -v id); do
+        if neutron port-show $p -f value | grep -qx binding_failed; then
```

Comment:
couldnt we move this test into neutron recipes, so that all cloud users would get notification of this error?